demos/storybook/stories/drawer/basic-config.stories.ts

```diff
@@ -0,0 +1,8 @@
+export const withBasicConfig = (): any => ({
+    template: `
```

Comment:
Empty story, is this intentional?

Reply:
I updated the story so that it is not empty. At this time we only have a drawer header. The full stories will be addressed in another issue.